src/software/networking/proto_multicast_sender.h

```diff
@@ -0,0 +1,47 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <string>
+
+template <class SendProto>
+class ProtoMulticastSender
+{
+   public:
+    /**
+     * Creates a ProtoMulticastSender that sends the SendProto over the network on the
+     * given address and port.
+     *
+     * @param io_service The io_service to use to service outgoing SendProto data
+     * @param ip_address The ip address of the multicast group to send data on
+     * (IPv4 in dotted decimal or IPv6 in hex string)
+     * example IPv4: 192.168.0.2
+     * example Ipv6: ff02::c3d0:42d2:bb8%wlp4s0 (the interface is usually specified after
```

Comment:
Nit: IPv6, not Ipv6 to keep it consistent with IPv4 above.

Reply:
fixed